### DISCUSSION TITLE
I recommend vps disk cleanup

### DISCUSSION THREAD

**Discussion by radiobellissima** [AzuraCast/AzuraCast#3878] [Q&A]
I have been using Azzurra for almost 1 year on my vps. I see that after the cancellation of several radios now the account of giga used is much higher than that of the active media at the moment. Can you recommend a safe cleanup command or something similar to check if anything is left of the already deleted radios.

Thanks
Marcello

**Comment by xewl** [comment]
If this is an up to date instance, then this might help:

If you go to the `Storage Locations` management via Administration, you can check if folders are assigned to any of the stations. If there is no station assigned to a folder, it is not in use.

(I'm not sure how "remnants" are handled tho')

**Comment by radiobellissima** [comment]
This has already been verified. I meant if it's possible for it to leave residue on the disc by deleting old stations and if there is a way to clean.

**Comment by Vaalyn** [comment]
You can look in the following directory if there is still a directory for the deleted station:
`/var/lib/docker/volumes/azuracast_station_data/_data`

This directory contains the station directories where stuff like media files, configs, etc are saved.
